models/endoscopic_tool_segmentation/docs/README.md

```diff
@@ -82,6 +92,14 @@ python -m monai.bundle run evaluating --meta_file configs/metadata.json --config
 python -m monai.bundle ckpt_export network_def --filepath models/model.ts --ckpt_file models/model.pt --meta_file configs/metadata.json --config_file configs/inference.json
 ```
 
+#### Export checkpoint to TensorRT based models with fp32 and fp16 precision:
+
+```
+python -m monai.bundle trt_export --net_id network_def --filepath models/model_trt_fp32.ts --ckpt_file models/model.pt --meta_file configs/metadata.json --config_file configs/inference.json --precision fp32
+
+python -m monai.bundle trt_export --net_id network_def --filepath models/model_trt_fp16.ts --ckpt_file models/model.pt --meta_file configs/metadata.json --config_file configs/inference.json --precision fp16
```

Comment:
I think no need to put 2 commands here, maybe just 1 command and explain that we can change the precision to FP16 by `--precision fp16`.

What do you think?

Thanks.

Reply:
Thank you for your advice. I have updated this part.